js/wp-seo-term-scraper.js

```diff
@@ -0,0 +1,318 @@
+/* global YoastSEO: true, wp, wpseoTermScraperL10n, ajaxurl, tinyMCE */
```

Comment:
I've created a separate issue for this: https://github.com/Yoast/wordpress-seo/issues/3066